Cargo.toml

```diff
@@ -4,9 +4,21 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+bytes = { version = "1.1", default-features = false }
 clap = { version = "4", features = ["derive"] }
-domain = "0.10.1"
+domain = { git = "https://github.com/NLnetLabs/domain", branch = "multiple-key-signing", features = [
+#domain = { path = "../domain2/", features = [
+    "openssl",
+    "ring",
+    "unstable-sign",
+    "unstable-validator",
+    "unstable-zonetree",
+    "zonefile",
+] }
 
 # for implementation of nsec3 hash until domain has it stabilized
 octseq = { version = "0.5.1", features = ["std"] }
 ring = { version = "0.17" }
+
+# for the signzone command
+openssl = { version = "0.10.57" } # required for RSASHA1 and 1024 bit key support (as ring doesn't support these)
```

Comment:
Won't `domain` include this for us?  Or should we be exporting it under `domain::dep`?